cmd/openvdc-executor/sshd.go

```diff
@@ -40,31 +35,25 @@ func NewSSHServer(provider hypervisor.HypervisorProvider, ctx context.Context) *
 
 type HostKeyGen func(rand io.Reader) (crypto.Signer, error)
 
-var KeyGenList = []HostKeyGen{
-	func(rand io.Reader) (crypto.Signer, error) {
-		_, priv, err := ed25519.GenerateKey(rand)
-		return priv, err
-	},
-	func(rand io.Reader) (crypto.Signer, error) {
-		return ecdsa.GenerateKey(elliptic.P521(), rand)
-	},
-	func(rand io.Reader) (crypto.Signer, error) {
-		return rsa.GenerateKey(rand, 2048)
-	},
+var KeyGenPathList = []string{
+	"/etc/openvdc/ssh/host_rsa_key",
+	"/etc/openvdc/ssh/host_ecdsa_key",
+	"/etc/openvdc/ssh/host_ed25519_key",
```

Comment:
I want here to set at the build time. Please refer``build.go`` where overwrite go top level variables.